docker-compose.yml

```
@@ -0,0 +1,93 @@
+version: "3.7"
+
+services:
+  enriched-data:
+    image: python:3.6
+    container_name: pc-enriched-data
```

```suggestion
    container_name: pcm-enriched-data
```
J'aurai différencié `pc` de `pcm` pour les noms des containers. Qu'en penses tu ?

---

oui pourquoi pas